client/my-sites/sharing/controller.js

```diff
@@ -23,17 +23,17 @@ import utils from 'lib/site/utils';
 
 const analyticsPageTitle = 'Sharing';
 
-export const layout = ( { contentComponent, path, store } ) => {
+export const layout = context => {
 	const site = sites().getSelectedSite();
 
 	if ( site && ! site.settings && utils.userCan( 'manage_options', site ) ) {
 		site.fetchSettings();
 	}
 
 	renderWithReduxStore(
-		createElement( Sharing, { contentComponent, path } ),
+		createElement( Sharing, { contentComponent: context.contentComponent, path: context.path } ),
```

Comment:
If anyone feels like this is more like how this should be written in Calypso:
```js
const { contentComponent, path } = context;

createElement( Sharing, { contentComponent, path } )
```
...lemme know. ;-)

Reply:
I'd prefer that :slightly_smiling_face:

Reply:
Hehe, updated. :-)